src/test/java/hw1/CalculatorDivideTest.java

```diff
@@ -0,0 +1,41 @@
+package hw1;
```

Comment:
Why is pull request for `hw2` contain `hw1` files?

Reply:
I've done a mistake, creating a new branch while I was working at the homework_1 branch in my local repository. Now the redundant files are removed from this branch